kolibri/core/content/utils/content_request.py

```diff
@@ -480,6 +480,7 @@ def process_download_request(download_request):
             baseurl=peer.base_url,
             node_ids=[download_request.contentnode_id],
         )
+        import_manager.download_request = download_request
```

Comment:
In the subclassed import manager, we should pass in the download request in the `__init__` function, rather than setting it externally.